src/Components/UI/Footer/Footer.css

```diff
@@ -8,12 +8,23 @@
 
 .footer-body h3 {
   font-weight: 400;
-  font-size: 0.9rem;
-  color: var(--text-dark);
+  font-size: 0.8rem;
```

Comment:
Suggestion: Could use font size unit as px if the font size is less than 1 rem.

Reply:
I'm using the default root font size of 16px, that's why didn't converted to px.